utils/kata-manager.sh

```diff
@@ -25,6 +25,7 @@ readonly containerd_project_url="https://github.com/${containerd_slug}"
 
 readonly kata_releases_url="https://api.github.com/repos/${kata_slug}/releases"
 readonly containerd_releases_url="https://api.github.com/repos/${containerd_slug}/releases"
+readonly containerd_io_releases_url="https//raw.githubusercontent.com/containerd/containerd.io/main/content/releases.md"
```

Comment:
Missing colon in URL which breaks the download:

```suggestion
readonly containerd_io_releases_url="https://raw.githubusercontent.com/containerd/containerd.io/main/content/releases.md"
```

Luckily manual testing caught this one ;)

```bash
+++ curl -fsSL https//raw.githubusercontent.com/containerd/containerd.io/main/content/releases.md
+++ grep -iE '\[d+.d+\].*| lts'
+++ grep -oE '\[[0-9]+.[0-9]+\]'
+++ grep -oE '[0-9]+.[0-9]+'
curl: (6) Could not resolve host: https
```

Reply:
Fixed, thanks!